pkg/tagger/collectors/ecs_extract.go

```diff
@@ -11,25 +11,33 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/tagger/utils"
 	"github.com/DataDog/datadog-agent/pkg/util/docker"
 	ecsutil "github.com/DataDog/datadog-agent/pkg/util/ecs"
+	"time"
 )
 
 func (c *ECSCollector) parseTasks(tasks_list ecsutil.TasksV1Response) ([]*TagInfo, error) {
 	var output []*TagInfo
+	now := time.Now()
 	for _, task := range tasks_list.Tasks {
+		// We only want to collect tasks without a STOPPED status
+		if task.KnownStatus == "STOPPED" {
+			continue
+		}
 		for _, container := range task.Containers {
-			tags := utils.NewTagList()
-			tags.AddLow("task_version", task.Version)
-			tags.AddLow("task_name", task.Family)
+			if c.expire.Update(container.DockerID, now) {
+				tags := utils.NewTagList()
```

Comment:
To make the intent clear, can you add a `// New container` comment line?

Reply:
👌